src/otx/algorithms/common/configs/configuration_enums.py

```diff
@@ -46,3 +46,14 @@ class BatchSizeAdaptType(ConfigurableEnum):
     NONE = "None"
     SAFE = "Safe"
     FULL = "Full"
+
+
+class InputSizePreset(ConfigurableEnum):
+    """Configurable input size preset."""
+
+    DEFAULT = "Default"
+    _256x256 = "256x256"
+    _384x384 = "384x384"
+    _512x512 = "512x512"
+    _768x768 = "768x768"
+    _1024x1024 = "1024x1024"
```

Comment:
Why don't you implement `__str__()` method to eliminate the under_bar from UX?

Reply:
In addition, we could add more options in this preset enum as I noted in other comments :)

Reply:
As far as I know, `ConfigurableEnum`, parent class of this, already have `__str()__` function, so string value is shown in UI.
I may be wrong, so please let me know if it is :)